scripts/test-in-lxd.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+set -eux
+
+IMAGE=$1
+SCRIPT=$2
+SERIES=${IMAGE##*:}
+TESTER=mini-iso-tools-$SERIES
+
+lxd init --auto
+
+lxc delete --force $TESTER || true
+
+if [ -z "$(lxc list -f csv -c n ^${TESTER}\$)" ]
+then
+    lxc launch $IMAGE $TESTER
+    lxc config device add $TESTER code disk source=`pwd` path=/src
+else
+    lxc start $TESTER
+fi
+
+lxc exec $TESTER -- sh -ec "
+    cd ~
+    sudo cp -a /src .
+    [ -d ~/src ]
+    "
+
+attempts=0
+while ! lxc file pull $TESTER/etc/resolv.conf - 2> /dev/null | grep -q ^nameserver; do
+    sleep 1
+    attempts=$((attempts+1))
+    if [ $attempts -gt 30 ]; then
```

Comment:
`-ge` for giving up after 30 seconds ; or you can start at `attempts=1` :)

Reply:
done